test/TensorFlow/control_flow.swift

```diff
@@ -78,7 +78,7 @@ public func weighPetOnlyDefault(pet: Pet) {
 
 // CHECK-LABEL: ---- ANALYSIS STATE FOR FUNCTION {{.*}}testCondBranch
 // CHECK:       bb0:
-// CHECK:       [Copy]    cond_br {{.*}}, bb1, bb2
+// CHECK:       [Copy]    cond_br {{.*}}, bb2, bb1
```

Comment:
do you know why BB numbers are changing for some tests (consider adding the explanation to PR description)?

Reply:
I believe this has to do with the fact that the deabstraction sees optimized SIL function for non-tensorflow convention functions.